framework/src/base/FEProblemBase.C

```diff
@@ -2785,7 +2784,7 @@ FEProblemBase::execute(const ExecFlagType & exec_type)
   // Set the current flag
   _current_execute_on_flag = exec_type;
   if (exec_type == EXEC_NONLINEAR)
-    _currently_computing_jacobian = true;
+    SubProblem::setCurrentlyComputingJacobian(true);
```

Comment:
This (and the other instances like it) can be left as it was (just doing `_currently_computing_jacobian = true;`) since we have access to that in this derived class.

Reply:
I have edited this.